.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ name: Publish release
 on:
   push:
     tags:
-      - "[0-9]+.[0-9]+.[0-9]+*"
+      - "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Why this change though? Is that to prevent the action from triggering automatically on e.g. `0.2.0a1`? If so, how can we release that?

We should probably allow a manual, trigger, right?

Reply:
> Why this change though? Is that to prevent the action from triggering automatically on e.g. `0.2.0a1`?

Yeah, exactly.

> If so, how can we release that?

With this PR as it stands, we would have no way to release that.  We'd have to wait until (possibly) there is a new version of the Action that supports the flag we want, which would allow zerover not to be marked as a pre-release.



> We should probably allow a manual, trigger, right?

I don't know enough about manual triggers to know if that would be the right solution (or how to allow it).  If there's a way to satisfy all our requirements immediately and without writing a separate `prerelease.yml` file, then I'd be very interested.

Reply:
I think adding a manual trigger option to `workflow_dispatch` should allow us to manually trigger the workflow. When doing so, a user is normally given a dropdown selection menu from which to choose a branch or tag on which to run the workflow. Cannot find the docs right now, but maybe that is enough of a pointer? Otherwise we can revisit this in a few days.